man/systemd.mount.xml

```diff
@@ -650,6 +650,19 @@
         <citerefentry><refentrytitle>systemd-system.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>.
         </para></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>GracefulOptions=</varname></term>
+
+        <listitem><para>Additional mount options that shall be appended to <varname>Options=</varname> if
+        supported by the local kernel. This may be used to define mount options that are optional and only
+        enabled on kernels that support them. Note that this is supported only for kernel mount options
+        (i.e. explicitly not for mount options implemented in userspace, such as those processed by
+        <command>/usr/bin/mount</command> itself, by FUSE or by mount helpers such as
+        <command>mount.nfs</command>.</para>
```

Comment:
Maybe, better to mention that this can be specified multiple times, unlike `Options=`.